packages/terra-overlay/tests/wdio/overlay-spec.js

```diff
@@ -0,0 +1,51 @@
+/* global browser, Terra */
+
+describe('Overlay', () => {
+  describe('Light Themed', () => {
+    beforeEach(() => browser.url('/#/tests/overlay-tests/light-themed'));
+
+    Terra.should.beAccessible();
+    Terra.should.matchScreenshot();
+    Terra.should.themeEachCustomProperty({
+      '--terra-overlay-content-color': 'red',
+      '--terra-overlay-content-padding-bottom': '500px',
+      '--terra-overlay-content-padding-left': '500px',
+      '--terra-overlay-content-padding-right': '500px',
+      '--terra-overlay-content-padding-top': '500px',
+      '--terra-overlay-light-background-color': 'green',
+      '--terra-overlay-light-background-image': 'linear-gradient(red, green)',
+      '--terra-overlay-light-opacity': '0.8',
+    });
+  });
+
+  describe('Dark Themed', () => {
+    beforeEach(() => browser.url('/#/tests/overlay-tests/dark-themed'));
+
+    Terra.should.beAccessible();
+    Terra.should.matchScreenshot();
+    Terra.should.themeEachCustomProperty({
+      '--terra-overlay-content-padding-bottom': '500px',
```

Comment:
We only need to check the themeable variables unique to the component. Given the content padding-left, content-padding-right, content-patting-top, and content-padding-bottom will update for any theme, lets only add this to one of these Terra.should.themeEachCustomProperty() test steps.